src/e2etests/requirements.txt

```
@@ -0,0 +1,3 @@
+pytest
+pytest-timeout
+requests
```

hmmm, code suggestion is the most top secret GitHub feature I've ever seen

```suggestion
pytest==5.4.1
pytest-timeout==1.3.4
requests==2.23.0
```